x-pack/plugins/cases/server/saved_object_types/cases.ts

```diff
@@ -179,6 +179,12 @@ export const createCaseSavedObjectType = (
       severity: {
         type: 'short',
       },
+      total_alerts: {
+        type: 'integer',
+      },
+      total_comment: {
```

Comment:
Is it a big job to change this to `total_comments`? I agree with your suggestion in the description of the PR that `total_comments` is a more logical name than `total_comment`, and would be consistent with the plural form used for `total_alerts`.

Reply:
It is trivial to change!

Initially, @cnasikas suggested keeping the names consistent with the existing ones, but I prefer `total_comments`.

Maybe there is something else I am missing?

Reply:
Yes, because the API returns `totalComment` for each case I suggested keeping it consistent. I agree `total_comments` is better. When we use it, we must remember to map it to `totalComment`.